media/transcoder/create_job_from_ad_hoc.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Transcoder sample for creating a job based on a supplied job config.
+
+Example usage:
+    python create_job_from_ad_hoc.py --project-id <project-id> --location <location> --input-uri <uri> --output-uri <uri>
+"""
+
+import argparse
+
+from google.cloud.video import transcoder_v1beta1
+from google.cloud.video.transcoder_v1beta1.services.transcoder_service import (
```

Comment:
Why is this in parentheses? This is unusual, so if there's a reason, please add a comment saying why to do it. Otherwise, please remove the parens.

Reply:
nox -s blacken is adding this